src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -32,27 +35,30 @@
 		public static class PWM {
```

Comment:
I thought we said ports would stay in RobotMap since it's a map of the robot? These are PWM ports.